magellan-legacy/src/test/java/com/wealthfront/magellan/ScreenTest.java

```diff
@@ -45,65 +42,57 @@ public void setUp() {
 
   @Test
   public void recreateView() {
-    View v = screen.recreateView(null, null);
+    View v = screen.recreateView(null);
```

Comment:
What calls `recreateView` if Screens are Navigable? It's not part of Navigable, which means it's not guaranteed to be called

Reply:
Same with `destroyView` below

Reply:
Removed these.